### PR TITLE
Add Nokia IXR H series

### PR DESCRIPTION
**Pull request by jbemmel**
* IXR H2 and IXR H4 devices
* DC power port type fixes

### CODE REVIEW

File: device-types/Nokia/7220-IXR-H2-128QSFP28.yaml [hunk 1]
```diff
@@ -0,0 +1,302 @@
+---
+manufacturer: Nokia
+model: 7220 IXR-H2 128QSFP28
+slug: nokia-7220-ixr-h2-128qsfp28
+part_number: 3HE16421AA
+u_height: 4
+is_full_depth: true  # 21.65"
+weight: 65.78
```
Comment (danner26) [inline device-types/Nokia/7220-IXR-H2-128QSFP28.yaml:8]: Do you have the `airflow` data point for this device or is it changeable based on the fan module?

File: device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
+manufacturer: Nokia
+model: 7220 IXR-H4 64QSFPDD 2SFP+
+slug: nokia-7220-ixr-h4-64qsfpdd-2sfpp
+part_number: 3HE17750AA
+u_height: 2
+is_full_depth: true  # 25.56"
+weight: 47.39
+weight_unit: lb
```
Comment (danner26) [inline device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml:9]: Do you have the `airflow` data point for this device or is it changeable based on the fan module?

File: device-types/Nokia/7220-IXR-H2-128QSFP28.yaml (hunk 1 above)
Comment (jbemmel) [reply device-types/Nokia/7220-IXR-H2-128QSFP28.yaml:8]: H2 supports both front-to-rear and rear-to-front FAN modules

File: device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml (hunk 2 above)
Comment (jbemmel) [reply device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml:9]: H4 only supports front-to-rear, so we could 'hardcode' it here

Comment (jbemmel) [reply device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml:10]: updated

**Comment by protonmarco** [comment]
@danner26 Is there any plan to merge this one?

**Review by danner26 — APPROVED** [review]

**Pull request merged**